openwhisk/README.md

```diff
@@ -0,0 +1,260 @@
+# OpenWhisk Runtimes for Knative
+
+This directory contains Tekton Pipelines which can be used to Build and Serve Knative compatible applications (i.e., OpenWhisk Actions) on Kubernetes.
+
+## Pre-requisites
+
+The general pre-requisites are as follows:
+- [x] Kubernetes v1.14.3
+- [x] kubectl
+- [x] Knative v0.5.0
+- [x] Tekton Pipeline
+- [x] Knative Client (OPTIONAL)
+
+Specifically, for development and testing on Mac OS, the following components and versions were used:
+
+- [x] [Docker Desktop for Mac Docker Community Edition 2.0.5.0 2019-06-11](https://docs.docker.com/docker-for-mac/edge-release-notes/) which includes:
+    - Docker 19.03.0
+    - Kubernetes 1.14.3
+- [x] [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (```brew install kubernetes-cli```)
+- [x] [Knative 0.5.0](https://github.com/knative/serving/releases/tag/v0.5.0) (which will install and configure istio-1.0.7 compatible resources)
+- [x] [Tekton Pipeline](https://github.com/tektoncd/pipeline/blob/master/docs/install.md#adding-the-tekton-pipelines)
+- [x] [Knative Client](https://storage.cloud.google.com/knative-nightly/client/latest/kn-darwin-amd64) (```mv client_latest_kn-darwin-amd64 /usr/local/bin/kn && chmod +x /usr/local/bin/kn```)
+
+### Docker Desktop Minimum Configuration
```

Comment:
I would like to keep it as part of main README since its still in experimental mode, these pre-requisites shows what version of Knative and Kubernetes we have tested with, and will update this as we test on recent version in future.

Reply:
+1, these docs are great. If they get too much longer I agree with @afrittoli it probably makes sense to move them elsewhere, but for now let's keep it here until we find out it's not a good idea. :)